README.md

```diff
@@ -55,6 +56,49 @@ See [pipelines/README.md](pipelines/README.md)
     * Set variable `enableUserWorkload` to `true`
   * Edit contents of [thanos-secret](acm/odh-core/acm-observability/secrets/thanos.yaml) file.
 
+### Gitea in cluster git server for GitOps workflow
+We are relying on the [gitea-operator](https://github.com/rhpds/gitea-operator) to manage the Gitea server installation in the cluster.  This will simplify the setup of Gitea so that we can create a minimal [Gitea](https://github.com/rhpds/gitea-operator#migrating-repositories-for-created-users) CR to configure and install the Gitea server.  The gitea-operator will be installed on the `odh-core` cluster as part of the ACM application rollout.
+
+1. Wait for the gitea-operator installation to complete and the `gitea.pfe-rhpds.com` CRD is available on the `odh-core` cluster
+   ```
+   $ oc get crd gitea.pfe.rhpds.com
+   NAME                  CREATED AT
+   gitea.pfe.rhpds.com   2023-08-25T03:00:13Z
+   ```
+
+1. Create the Gitea CustomResource to deploy the server with an admin user
+   ```
+   ---
+   apiVersion: v1
```

Comment:
Are you sure about including the whole resource file in the README? I personally like Ger's approach of using a Make recipe more if parameters/placeholders are involved. I'd also prefer just including this as a file in the repo and telling the reader to `oc create -f` it.

If including it in the README in necessary I'd prefer if you at least provided an easy copy/paste command e.g.:
```bash
 cat <<EOF | oc apply -f -
 apiVersion: v1
 ...
 EOF
```

Reply:
I was thinking the same thing which is why I created #116 to follow-up with those changes. I'm leaving this as is since the install of the gitea-operator dependency can be handled by ACM right now.

> If including it in the README in necessary I'd prefer if you at least provided an easy copy/paste command e.g.:
Seems reasonable